service/build.gradle

```diff
@@ -221,14 +234,16 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'send-letter-client', version: '2.5.0'
   compile group: 'uk.gov.service.notify', name: 'notifications-java-client', version: '3.15.1-RELEASE'
 
-  compile group: 'com.google.guava', name: 'guava', version: '29.0-jre'
-  compile group: 'commons-io', name: 'commons-io', version: '2.7'
+  implementation group: 'com.google.guava', name: 'guava', version: '29.0-jre'
+  implementation group: 'commons-io', name: 'commons-io', version: '2.7'
 
-  compile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
+  implementation group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
 
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-mail'
 
-  testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot) {
+  testImplementation group: 'org.apache.pdfbox', name: 'pdfbox', version: '2.0.19'
+
+  testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test') {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.mockito', module: 'mockito-core'
```

Comment:
Random question but do you know why we are excluding mockito here?
Is there something wrong with the version that is included with `spring boot starter test` that means 3.0.0 is wanted?

Reply:
hmmm, not sure but looking through the git history it was done as part of a gradle plugin update. Here is the commit hash `914d8e5cb5db6ce5ca474516f17b4fca2943a87e`